Yoast/wpseo-news
# Add term exclusion for all taxonomies connected to enabled post types

## Summary

This PR can be summarized in the following changelog entry:

* Allows term based exclusion for all taxonomies connected to enabled post types.

## Relevant technical choices:

* Moved the `catexclude_` options to a `term_exclude_<taxonomy>_<slug>` option for better portability.

## Test instructions

This PR can be tested by following these steps:

* Go to the Yoast SEO News admin page.
* Enable at least one custom post type, and hit save.
* Select a couple of (custom) taxonomy terms.
* Create a post and add one of those taxonomy terms you just selected.
* Create another post and do _not_ add one of those taxonomy terms you just selected.
* See only the second post in the XML news sitemap.

Be sure to also test the upgrade path for the old settings to the new settings.

Fixes #414 
Fixes #415
